example/UpdateUserLiveTest.php

```diff
@@ -18,24 +18,40 @@
 
 use CBS\SmarterU\Client;
 use CBS\SmarterU\DataTypes\User;
+use CBS\SmarterU\DataTypes\Timezone;
+use CBS\SmarterU\Exceptions\SmarterUException;
 
 /**
  * This script contains a live test for Client::updateUser. It was used to
  * intentionally trigger errors and see how the API responds.
  */
-$accountKey = ''; //insert key here before running
-$userKey = ''; //insert key here before running
+$accountKey = '87CD3F946F81B62242AC4B5E4DC8F59F'; //insert key here before running
+$userKey = 'st^cyxyxoj6okog69ih4is983qufylzhe8*uf8c8'; //insert key here before running
```

Comment:
Please tel me these are not your keys... Maybe use the environment variable thing you use in `example/CreateUserLiveTest.php`

Reply:
I updated it to use `getenv().

Reply:
Reaching out to SmarterU to see about cycling our API keys since I screwed up so bad here.  @tomegantcs  and I had a side conversation about what else to do with this. We could replace these lines with `getenv()` as I did in a few spots now, or we could parameterize these scripts with `getopt()`.